src/debug.h

```diff
@@ -25,11 +25,6 @@
 
 #include <QDebug>
 
-// DEBUG_COMMHISTORY is defined in debug build (CONFIG += debug)
-#ifdef DEBUG_COMMHISTORY
```

Comment:
`DEBUG_COMMHISTORY` is used in a few other places. Undefining it takes out some debug trace at compile time.